Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -136,11 +141,22 @@ internal final class AddNewCardViewController: UIViewController,
     _ = self.zipcodeView
       |> \.autocapitalizationType .~ .allCharacters
       |> \.returnKeyType .~ .done
+
+    _ = self.reusableCardLabel
+      |> \.text %~ { _ in Strings.Remember_this_card() }
+
+    _ = self.reusableCardStackView
+      |> \.alignment .~ .center
+
+    _ = self.reusableCardSwitch
+      |> baseSwitchControlStyle
   }
 
   override func bindViewModel() {
     super.bindViewModel()
 
+    self.reusableCardStackViewContainer.rac.hidden = self.viewModel.outputs.reusableCardSwitchIsHidden
```

Comment:
Should we order this alphabetically (so before the `saveButtonIsEnabled` signal)?